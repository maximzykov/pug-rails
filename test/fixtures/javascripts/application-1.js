//= require templates/jade.jst.jade
//= require templates/pug.jst.pug
