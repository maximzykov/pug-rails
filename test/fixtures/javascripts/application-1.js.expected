(function() { this.JST || (this.JST = {}); this.JST["templates/jade"] = (function(jade) { function template(locals) {
  var buf = [];
  var jade_mixins = {};
  var jade_interp;

  buf.push("<div>Hello, Jade!</div>");;return buf.join("");
  }; return template; }).call(this, jade);;
}).call(this);
(function() { this.JST || (this.JST = {}); this.JST["templates/pug"] = (function() { function template(locals) {var pug_html = "", pug_mixins = {}, pug_interp;pug_html = pug_html + "\u003Cdiv\u003EHello, Pug!\u003C\u002Fdiv\u003E";;return pug_html;}; return template; }).call(this);;
}).call(this);
